repair/row_level.cc

```diff
@@ -1169,6 +1169,10 @@ class repair_meta {
         } else {
             return;
         }
+        flush_rows(_schema, _working_row_buf, _repair_writer, _db, _sys_dist_ks, _view_update_generator);
+    }
+
+    static void flush_rows(schema_ptr _schema, std::list<repair_row>& _working_row_buf, lw_shared_ptr<repair_writer>& _repair_writer, seastar::sharded<replica::database>& _db, seastar::sharded<db::system_distributed_keyspace>& _sys_dist_ks, seastar::sharded<db::view::view_update_generator>& _view_update_generator) {
```

Comment:
Let's say this patch introduces a new helper function flush_rows in the commit log and you want to make the helper a free function later. It took me a while to figure out how you split ;-)

Reply:
The diff is surprisingly dense, I can update the commit comment to make it more easily digestable.